src/pages/governance/_components/VotingResult.tsx

```diff
@@ -90,92 +90,82 @@ export function VotingResult({
                 "mb-6 lg:pb-[22px] lg:pr-0 md:pr-8 md:pb-0 pb-[22px] lg:w-full w-full md:w-1/2 lg:border-b-[0.5px] md:border-b-0 border-b-[0.5px] dark:border-dark-gray-300"
               )}
             >
-              <div className="flex flex-col gap-y-2 w-full">
-                <span className="md:text-lg text-sm font-semibold text-gray-900 dark:text-dark-gray-900">
-                  Yes
-                </span>
-                <Progress
-                  value={percYes.toNumber()}
-                  containerClass="bg-gray-100 dark:bg-dark-gray-200"
-                  contentClass="bg-gray-300 dark:bg-dark-gray-400"
-                />
-                <div className="flex flex-row">
-                  <span
-                    className={classNames(
-                      "grow text-gray-600 dark:text-dark-gray-600 text-sm"
-                    )}
-                  >
-                    {percYes.toFixed(2)}%
+              <div className="flex flex-col w-full">
+                <div className="flex flex-row mb-1">
+                  <span className="md:text-base text-sm font-semibold text-gray-900 dark:text-dark-gray-900 grow">
```

Comment:
check the spacing with the progress bar

Reply:
checked that the padding between the bar and the labels are 10px evenly updated to use gap-y-[10px] removed the extra spacing for the threshold line